bench_runner/scripts/should_run.py

```diff
@@ -49,6 +50,8 @@ def main(
     flags = []
     if tier2:
         flags.extend(["PYTHON_UOPS"])
+    if jit:
+        flags.extend(["JIT"])
```

Comment:
You can just append here:

```suggestion
    if tier2:
        flags.append("PYTHON_UOPS")
    if jit:
        flags.append("JIT")
```